docs/developer-docs/multi-chain/ethereum/evm-rpc/overview.mdx

```diff
@@ -0,0 +1,85 @@
+
+- **RPC service**: An IPv6 service such as Alchemy, Gateway.fm, or CloudFlare that offers access to the Ethereum JSON-RPC API over HTTP. Note that other EVM blockchains are often covered by such a JSON-RPC API as well.
+
+- **network**: An EVM blockchain such as the Ethereum mainnet or Sepolia testnet.
+
+- **chain id**: A well-known unique numeric identifier for an EVM network.
+
+-**provider**: A specific JSON-RPC service registered in the EVM RPC canister. Every chain id for a particular service requires a different provider and typically also requires a different API key. Multiple providers can be registered for a service / chain id combination.
+
+## Management of API keys
+
+RPC providers often require API keys be used to authenticate RPC requests. Authenticated requests help mitigate potential DoS attacks that exhaust the request rate limits set by RPC providers. Using an API key can come with challenges for developers, such as:
+
+- API keys for RPC providers often require a paid subscription and/or compute credits beyond a certain number of requests per month.
+
+- For developers who want to build a quick prototype, it may not be worth procuring multiple API keys until later in development.
+
+- When a dapp developer controls an API key, the dapp is considered less decentralized.
+
+- Storing API keys securely is not supported at the platform level and requires developers to use their own strategies for protecting their keys, creating an additional complexity for developers who want to use reproducible builds. Personal keys are also visible to all nodes on the ICP subnet. [Learn more about authentication](how-it-works.mdx#authentication).
+
+To provide a solution for developers, the production EVM RPC manages the API keys for each RPC provider on behalf of the developer. If a developer wants to use their own API keys, however, they can [pass their own keys](evm-rpc-canister.mdx#replacing-api-keys) to the EVM RPC canister. Test variations of the EVM RPC canister do not use API keys.
+
+## Costs
+
+Calls made to the EVM RPC canister cost [cycles](/docs/current/developer-docs/getting-started/cycles/cycles-faucet). The cost per request is generally similar to making the equivalent [HTTPS outcalls](https://internetcomputer.org/docs/current/references/https-outcalls-how-it-works#pricing) in your own canister.
+
+Learn more about the [costs of EVM RPC requests](costs.mdx).
+
+## Canisters
+
+- Production canisters with API keys managed by DFINITY:
+
+    - Fiduciary subnet (28 nodes): [`7hfb6-caaaa-aaaar-qadga-cai`](https://dashboard.internetcomputer.org/canister/7hfb6-caaaa-aaaar-qadga-cai)
+
+- Test canisters with no API keys:
+
+    -  Standard subnet (13 nodes): [`a6d44-nyaaa-aaaap-abp7q-cai`](https://dashboard.internetcomputer.org/canister/a6d44-nyaaa-aaaap-abp7q-cai)
+
+    -  Fiduciary subnet (28 nodes): [`xhcuo-6yaaa-aaaar-qacqq-cai`](https://dashboard.internetcomputer.org/canister/xhcuo-6yaaa-aaaar-qacqq-cai)
```

Comment:
its not entirely clear that the canisters mentioned here are different evm-rpc-canister deployments imo

Reply:
Would `## Canisters` -> `## Deployed canisters` help clarify this?

Reply:
Revised to `Deployed EVM RPC canisters`